javaparser-core/src/main/java/com/github/javaparser/ast/expr/MethodCallExpr.java

```diff
@@ -65,6 +65,14 @@ public MethodCallExpr(final Expression scope, final String name) {
         this(null, scope, new NodeList<>(), new SimpleName(name), new NodeList<>());
     }
 
+    public MethodCallExpr(final Expression scope, final SimpleName name) {
```

Comment:
Looks good. Eventually we could remove some of these constructors, but for now adding these two seems the way to go